content/applications/sales/point_of_sale/receipts_invoices.rst

```diff
@@ -32,6 +32,8 @@ When you use it, you can then reprint your last receipt.
 
 .. image:: receipts_invoices/reprint03.png
 
+.. _receipts/invoices:
```

Comment:
just as a comment:
I suggest another anchor, since this page filenames include both receipts and invoices:
`receipts-invoices/invoices
`
or even
`pos-invoices`